index.src.html

```diff
@@ -631,6 +634,8 @@ <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
                   // Insert some code here to fall back to a basic login form.
                 }
               });
+            // Alternatively, the website could just copy the credential data into the
+            // form and call submit() on the form.
```

Comment:
(Regardless of focus, we should have an entirely `form.submit()`-based example, as that fits a number of use cases.)